contest/static/contest/js/jquery.quicksearch.js

```diff
@@ -0,0 +1,181 @@
+(function($, window, document, undefined) {
```

Comment:
The same issue with my previous comment.

If we can get javascript library online, that wilil be better.
I don't think the code need to put up on this repository.
(or use bower to manage such kind of library locally, http://blog.wu-boy.com/2013/01/bower-is-a-package-manager-for-the-web/)